externals/kyuubi-flink-sql-engine/pom.xml

```diff
@@ -165,7 +165,7 @@
               <include>com.google.guava:failureaccess</include>
               <include>com.google.guava:guava</include>
               <include>commons-codec:commons-codec</include>
-              <include>io.netty:netty-all</include>
+              <include>io.netty:*</include>
```

Comment:
why do we have netty here？

Reply:
the artifact id of netty changed since netty does not provide a fat package

Reply:
where do we use netty，especially between server and engine

Reply:
The obvious place is `KyuubiConf`, it uses `NettyUtils.MAX_NETTY_THREADS`.

The netty dependency is introduced for mysql frontend.  So I think we can remove the netty dependency at engine side ? @pan3793

Reply:
we can remove it from common

Reply:
in this pr，let's do it your way

Reply:
Move `NettyUtils` to `kyuubi-server` module makes sense to me.